explanation/service-endpoint-encryption.rst

```diff
@@ -10,8 +10,7 @@ OpenStack supports enabling TLS via the Traefik application, which is the
 ingress point for all service endpoints.
 
 .. note::
-   Currently, only the TLS CA feature method is supported. This feature only
-   works with certificates signed by an external Certificate Authority.
+   Currently, only the TLS CA and TLS Vault feature methods are supported. TLS CA only works with certificates signed by an external Certificate Authority.
 
 TLS CA feature
```

Comment:
[* suggestion *] Instead of repeating "feature" after every feature, I suggest naming just the feature, e.g. `TLS CA` and `TLS Vault`